NEWS.md

```diff
@@ -1,3 +1,7 @@
+# naomi.options 1.3.0
+
+* Update default for 2024/2025 HIV estimates
```

Comment:
Can you add to this the details of specific updates that you put in the PR notes? That may be helpful to refer back to next year.